### DISCUSSION TITLE
Improve waveform view with very basic editing and zooming capabilities

### DISCUSSION THREAD

**Discussion by helgoboss** [helgoboss/helgobox#1012] [Ideas]
At the moment, it doesn't allow any interactions

**Comment by Akognito** [comment]
++++++++++++

**Comment by ThoughtPhotography** [comment]
Content preview is easily the biggest improvement in Bitwig's Session View over Live's
![2 - CLIP ANATOMY - v2 4](https://github.com/user-attachments/assets/8c21fcb6-4ab1-49e1-94f1-aa0f6b46cefd)

**Comment by declanhalpin** [comment]
PLEASE PLEASE PLEASE!  For me, Ctrl+wheel up/down would be perfect.  The little magnifying glasses (like in Reaper) are terrible.

**Comment by Qsonic72** [comment]
+ simpel, better start and endpoint selection
